data/saas/dataset/shopify_dataset.yml

```diff
@@ -1539,11 +1539,11 @@ dataset:
             fidesops_meta:
               data_type: integer
           - name: author
-            data_categories: [user.name]
+            data_categories: [system.operations]
```

Comment:
@galvana does a Dataset update also require a version bump in the config?

Reply:
Hey @SteveDMurphy, yes it does, whenever Fides sees the version bump it will update both the dataset and config. So in our case, we can bump the version even if it's just a dataset change